tutorials/napps/index.rst

```diff
@@ -1,17 +1,7 @@
-:orphan:
-
-.. _tutorial-napps:
-
 Network Applications
 ********************
 
 Here you will find tutorials related to Kytos Network Applications.
 
 If you have any doubt, do not hesitate to ask the Kytos community at our IRC
-Channel #kytos at freenode.net.
-
-.. toctree::
-  :maxdepth: 1
-  :hidden:
-
-  create_your_napp
+Channel #kytos at freenode.net.
```

Comment:
This is outdated. Please update to https://kytos.slack.com/

Reply:
It's up to date now. Thanks @hdiogenes!